pkg/utils/utils.go

```diff
@@ -110,3 +110,14 @@ func ToDNS1123Subdomain(name string) string {
 	}
 	return name
 }
+
+// GenerateFixedName generates a fixed name for the given owning resource and human readable prefix.
```

Comment:
If this is going to merge before #1563 (which seems likely), then pull over `utils_test.go` too.

Reply:
Done, thanks!